### PR TITLE
periodic-installer-s390x job changes

### PR DESCRIPTION
**Pull request by LakshmiRavichandran1**

### CODE REVIEW

File: ci-operator/jobs/openshift/multiarch/openshift-multiarch-release-4.8-periodics.yaml [hunk 1]
```diff
@@ -1357,3 +1357,121 @@ periodics:
     - name: release-pull-secret
       secret:
         secretName: ci-pull-credentials
+- agent: kubernetes
```
Comment (deepsm007) [inline ci-operator/jobs/openshift/multiarch/openshift-multiarch-release-4.8-periodics.yaml:1360]: Move this job to https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift/release/openshift-release-release-4.8-periodics.yaml file

Comment (LakshmiRavichandran1) [reply ci-operator/jobs/openshift/multiarch/openshift-multiarch-release-4.8-periodics.yaml]: made necessary changes in the commit - https://github.com/openshift/release/pull/18849/commits/ab4dce3eee00af72e3616f444dab5c26c51b7b47

**Review by petr-muller — APPROVED** [review]

**Pull request merged**